src/main/java/org/cyclops/integratedtunnels/part/aspect/TunnelAspectWriteBuilders.java

```diff
@@ -77,6 +80,14 @@
 
     public static final IAspectPropertyTypeInstance<ValueTypeBoolean, ValueTypeBoolean.ValueBoolean> PROP_BLACKLIST =
             new AspectPropertyTypeInstance<>(ValueTypes.BOOLEAN, "aspect.aspecttypes.integratedtunnels.boolean.blacklist.name");
+    public static final IAspectPropertyTypeInstance<ValueTypeInteger, ValueTypeInteger.ValueInteger> PROP_CHANNEL =
```

Comment:
Like in ID, I feel like this should not be needed. If we keep it though, we should remove this anyways and use the property defined in ID instead.